src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -36,6 +36,7 @@ public HelpWindow(Stage root) {
         super(FXML, root);
         helpMessage.setText(HELP_MESSAGE);
         root.sizeToScene();
+        root.setResizable(false);
```

Comment:
This can be set in the FXML file so let's do it there so that all UI layout stuff is kept in one file.